### ISSUE TITLE
Unknown filter type: default - phalcon_3.3.0

### ISSUE THREAD

**Issue by kowach** [phalcon/cphalcon#13244]
After update from 3.2.4-1 => 3.3.0-1 "default" filter is not recognized any more.

Unknown filter type in .volt at line (x) => `{{ myvar|default('some var') }}`

OS: Ubuntu Mate 16.04
PHP 7.0.22-0ubuntu0.16.04.1 (cli) ( NTS )

Package: php7.0-phalcon
Version: 3.3.0-1
Build Date: Dec 24 2017 10:57:53

Full exception:
```
Unknown filter type in /../index.volt on line 8
#0 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->resolveFilter(Array, '$preparedParams')
#1 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->expression(Array)
#2 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->expression(Array)
#3 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->compileEcho(Array)
#4 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->_statementList(Array, false)
#5 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->_compileSource('<script>\n    "u...', false)
#6 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->compileFile('/var/.../...', '/.../...', false)
#7 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->compile('/var/../...')
#8 [internal function]: Phalcon\Mvc\View\Engine\Volt->render('/.../...', Array, true)
#9 [internal function]: Phalcon\Mvc\View->_engineRender(Array, 'default/index', true, true, NULL)
#10 [internal function]: Phalcon\Mvc\View->render('mailer', 'index')
#11 /var/.../index.php(43): Phalcon\Mvc\Application->handle()
#12 {main}

```
index.volt (at line 8)
```
{{ preparedParams|default([])|json_encode }}
```

**Comment by sergeyklay** [comment]
Fixed in the `3.3.x` branch. Feel free to open a new issue if the problem appears again. Thank you for the report, and for helping us make Phalcon better.

Refs: https://github.com/phalcon/cphalcon/issues/13242

**Comment by fendorio** [comment]
Anyone know who is responsible for managing Phalcon releases on homebrew? Currently the latest version available is 3.3.0, which of course, contains this bug..

**Comment by SMillerDev** [comment]
Homebrew doesn't have the concept of people managing releases. Everyone is welcome to submit a pull request to include a new version.
